pkgs/applications/misc/tdrop/default.nix

```diff
@@ -1,31 +1,41 @@
 { stdenv, lib, fetchFromGitHub, makeWrapper
-, xwininfo, xdotool, xprop }:
+, xwininfo, xdotool, xprop, gawk, coreutils
+, gnugrep, procps-ng }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
```

Comment:
From quickly eyeballing, there don't seem to be any changes that require a recursive attrset? (Though I may have missed it.)